events/2023-10-in-person/index.qmd

```diff
@@ -80,4 +80,126 @@ some topics and less on others.
 
-## Minutes
\ No newline at end of file
+## Minutes
+
+### Workshop with Mjølner
+
+I'll be creating a blog post about this experience, so won't add much here expect some misc items.
+
+- Comments from them:
+    - What about backups of the data? In the Docker container or somewhere else?
+    - Is it something to add to docs or is it a feature?
+    - Transferring data to servers is a lot more difficult than it seems, so best have that as many versions in the future (if at all)
+    - A lot of language we used was about "limiting access", but it might be useful to use language on "enabling access"
+    - Be careful about store logs too much, they can quickly bloat
+    - How to handle backward compatibility? Part of testing to include tests of demo?
+- We need to get some fake/public data or something to use for demo and test purposes
+
+
+### Workshop debrief:
+
+- General comments
+    - Obvious they have skill in frontend dev, based on what they showed us
+    - Can't judge what their security capabilities or backend capability are though
+    - They had good process/flow for guiding us, they kept good pace and kept on time
+    - They were very professional and friendly to work with, good seniority/experience for them
+    - Obvious they had read up on what we were doing and were prepared
+    - Helped us to focus down on what things would be version 1 vs later versions, as well as identifying hotspots and things to move on with
+    - It was really useful to see how they work as well, though maybe not loving the purely scrum style they use
+- Hiring them and what/how they can work with us
+    - Could either hire them after we have a core functional box or hire them to help use develop our process and workflow (getting started, project management, teamwork practices, "how to work" rather than "what to work on")
+    - If hired for process, we'll need to give them clear requirements for what we want from them
+        - We might need some time to sit down and define the requirements for designing the how to work together
+    - [X] Action: To contact them about hiring them, what are the next steps? How them charge
+    - [ ] Action: Need to spend some time writing doc on how to work together, the process. This could be the next “focus stage”, on starting to write out a design document
+    - [ ] Action: Make a detailed list of what outputs we would like from them
+        - E.g. training material or documentation on working together and their process
+        - E.g. finalize the design doc (maybe another review of it)?
+- Things we could take from them
+    - Their approach to visualizing things (e.g. for instance with the vertical lines)
+        - For virtual settings: Look into online tools that help with doing the visualize
```

Comment:
Thought: They said that they would use Figma to digitalise the event storming boards. Maybe we could as well? Figma allows for real-time collaboration (not sure whether it's a free feature though): https://www.figma.com/collaboration/